test/SeePIDConstants.cpp

```diff
@@ -31,7 +27,7 @@ unittest(TestVerticalScroll) {
   tc->loop();
   assertEqual("SeePIDConstants", tc->stateName());
 
-  // during the delay we cycle through kp,ki,kd, and ph slope
+  // during the delay we cycle through kp,ki, and kd
   assertEqual("Kp: 10001.0     ", display->getLines().at(0));
```

Comment:
I thought the view was going to just show kp and ki, then show kd, then go back to kp and ki.

Reply:
What we discussed was a loop of the following:
"Kp / Ki"  then "Ki / Kd" then "Kd / blank"
An alternate would be "Kp / Ki" then "Kd / blank" (no need to give Ki double listing).